File: java/pca-model/src/main/java/com/sdl/web/pca/client/contentmodel/pagemodeldata/ContextData.java
package com.sdl.web.pca.client.contentmodel.pagemodeldata;

import com.sdl.web.pca.client.contentmodel.ClaimValue;
import com.sdl.web.pca.client.contentmodel.IContextData;

import java.util.List;

public class ContextData implements IContextData {

    private List<ClaimValue> ClaimValues;

    @Override
    public List<ClaimValue> getClaimValues() {
        return ClaimValues;
    }

    @Override
    public void setClaimValues(List<ClaimValue> claimValues) {
        ClaimValues = claimValues;
    }
}

File: net/src/Sdl.Tridion.Api.Client/ContentModel/ContextData.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Sdl.Tridion.Api.Client.ContentModel
{
    /// <summary>
    /// Context Data
    /// </summary>
    public class ContextData : IContextData
    {
        public ContextData()
        {
        }

        public ContextData(IContextData copy)
        {
            ClaimValues = new List<ClaimValue>(copy.ClaimValues);    
        }

        /// <summary>
        /// List of claim values to pass to query.
        /// </summary>
        public List<ClaimValue> ClaimValues { get; set; } = new List<ClaimValue>();
    }  
}
